packages/boot/README.md

```diff
@@ -0,0 +1,88 @@
+# @loopback/boot
+
+A collection of Booters for LoopBack Applications
+
+# Overview
+
+A Booter is a Class that can be bound to an Application and is called
+to perform a task before the Application is started. A Booter may have multiple
+phases to complete its task.
+
+An example task of a Booter may be to discover and bind all artifacts of a
+given type.
+
+BootComponent is responsible for handling Booter artifacts and running the
+phases. It must be added to an `Application` to use `await app.booter()`
+or `await app.boot()`.
+
+## Installation
+
+```shell
+$ npm i @loopback/boot
+```
+
+## Basic Use
+
+```ts
+import {Application} from '@loopback/core';
+import {ControllerBooter, BootComponent} from '@loopback/boot';
+const app = new Application({components:[BootComponent]});
```

Comment:
This won't work after https://github.com/strongloop/loopback-next/issues/742 is done. I am proposing to consider `components` config property as deprecated and stop using it in new code and examples, so that we have less things to fix as part of #742.

```ts
const app = new Application();
app.component(BootComponent);
// etc.
```